packages/fxa-auth-server/lib/payments/stripe.js

```diff
@@ -300,7 +300,7 @@ class StripeHelper {
         {
           customer: customerId,
         },
-        { idempotencyKey }
+        { idempotencyKey: `pma-${idempotencyKey}` }
```

Comment:
What's the significance of these prefixes?

Reply:
The prefix is just an arbitrary string to make each of the keys unique. Reusing the same idempotency key for both of those API calls resulted in the second one failing. I forget the specific error message, but the gist was Stripe wants distinct idempotency keys for different API calls.